pkg/config/package.go

```diff
@@ -294,6 +294,7 @@ type Param struct {
 	Installed                         bool
 	PolicyConfigFilePaths             []string
 	Commands                          []string
+	VacuumDays                        *int // When defined, vacuuming is enabled
```

Comment:
```suggestion
	VacuumDays                        int // When defined, vacuuming is enabled
```

I don't think this should be a pointer.

Reply:
VacuumDays is now an integer and not a pointer